server/controllers/routes/user/student/getAlumniCohort.js

```diff
@@ -0,0 +1,28 @@
+const { getSpecificAlumni } = require('../../../../database/queries');
+const { getCohortQuery } = require('../../../../database/queries');
+
+const getAlumniCohort = async (req, res, next) => {
+  try {
+    const { cohortId } = req.params;
+    if (cohortId > 0) {
+      const checkCohort = await getCohortQuery(cohortId);
```

Comment:
I think no need for this query as if you used `getSpecificAlumni(cohortId)` with an `id` that doesn't exist then you can check the result and decide if there is info or not

Reply:
yes you are right there is no needed of this query if cohortId doesn't exist  it will send an empty array.
![58745555](https://user-images.githubusercontent.com/55782435/78453309-80ba7a80-7699-11ea-972f-e324be2a473a.png)
i will modify it, thank you :100:

Reply:
yes you are right there is no needed of this query if cohortId doesn't exist  it will send an empty array.
![58745555](https://user-images.githubusercontent.com/55782435/78453309-80ba7a80-7699-11ea-972f-e324be2a473a.png)
i will modify it, thank you :100: